frontend/packages/console-dynamic-plugin-sdk/src/runtime/plugin-utils.ts

```diff
@@ -1,3 +1,3 @@
-import { StandardConsolePluginManifest } from '../build-types';
+import { ConsolePluginManifest } from '../build-types';
 
-export const getPluginID = (m: StandardConsolePluginManifest) => `${m.name}@${m.version}`;
+export const getPluginID = (m: ConsolePluginManifest) => `${m.name}@${m.version}`;
```

Comment:
Note that this code will eventually get removed since in OpenShift plugin SDK (core runtime lib) we identify plugins only by their names, i.e. `{name}@{version}` plugin ID part is legacy due to Console plugin loader code.